dandischema/consts.py

```diff
@@ -23,6 +23,8 @@
 
 # This allows multiple schemas for validation, whereas target schemas focus on
 # migration.
+# We use simple sorted() here to avoid circular imports, but use
+# utils.sort_versions() if to output to the user
 ALLOWED_VALIDATION_SCHEMAS = sorted(
     set(ALLOWED_INPUT_SCHEMAS).union(ALLOWED_TARGET_SCHEMAS)
 )
```

Comment:
```suggestion
ALLOWED_VALIDATION_SCHEMAS = sorted(
    set(ALLOWED_INPUT_SCHEMAS).union(ALLOWED_TARGET_SCHEMAS), key=Version
)
```

I understand the need to avoid circular definition, and not use `sort_versions` here. However, I would like to have `ALLOWED_VALIDATION_SCHEMAS` sorted correctly in the first place. I think it will be hard to remember or enforce everyone to remember to resorting `ALLOWED_VALIDATION_SCHEMAS` when it is output to the user.

With this solution, we don't even define the `sort_versions()` function, but the function is a one line function, and  the behavior of `packaging.version.Version` is well documented.

The other less preferable solution to me is to define the `sort_versions()` function in this file, `consts.py`. However, that will contradict with the name of this file. We can change the name to `_sort_versions()` though.

Reply:
cute, let's do, but need to import `from packaging.version import Version as _Version` so we do not pollute the namespace with imports.